app/templates/application.hbs

```diff
@@ -1,82 +1,84 @@
-{{#x-app
-  active=this.active
-  isDragging=this.isDragging
-}}
-  {{#if isEmberApplication}}
-    <div class="split">
-      <div class="split__panel">
-        {{!-- Begin main --}}
-        <div class="split split--main">
-          {{#draggable-column
-            width=this.navWidth
-            classes="split__panel split__panel--sidebar-1"
-            setIsDragging=(action "setIsDragging")
-          }}
-            <div class="split__panel__hd">
-              {{app-picker}}
-            </div>
-            <div class="split__panel__bd">
-              <SideNav @deprecationCount={{this.deprecationCount}} />
-            </div>
-            <div class="split__panel__ft">
-              <a target="_blank" href="https://github.com/emberjs/ember-inspector/issues">
-                Submit an Issue
-              </a>
-            </div>
-          {{/draggable-column}}
+<div class="ember-view">
+  {{#x-app
+    active=this.active
+    isDragging=this.isDragging
+  }}
+    {{#if isEmberApplication}}
```

Comment:
Nit pick: `this. isEmberApplication`

Reply:
Definitely do need to do changes like this, but that was not meant to be part of this PR. This PR is just running the Octane blueprints and fixing issues.

Reply:
:+1: